src/main/resources/edu/cornell/kfs/sys/businessobject/datadictionary/UserProcurementProfile.xml

```diff
@@ -149,19 +162,23 @@
     
   </bean>
   
-        <bean id="UserProcurementProfile-profileUser.principalName" parent="UserProcurementProfile-profileUser.principalName-parentBean"/>
+  <bean id="UserProcurementProfile-profileUser.principalName" parent="UserProcurementProfile-profileUser.principalName-parentBean"/>
 
-  <dd:boAttributeRef id="UserProcurementProfile-profileUser.principalName-parentBean" parent="PersonImpl-principalName" abstract="true" 
-  attributeName="profileUser.principalName" label="User Principal Name" shortLabel="Principal Name" required="false">
-    <dd:user universalIdAttribute="principalId" userObjectAttribute="profileUser"/>

```

Comment:
This "dd:user" tag doesn't seem to have been translated into the equivalent Spring XML by this rewrite. Can you please add the related Spring XML for this if necessary? I believe this would translate to a "KualiUserControlDefinition" bean.